packages/kit-react-native/src/lists.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import { FlatList, FlatListProps, SectionList, SectionListProps } from 'react-native'
+
+type Item = { key: string; label: React.ReactElement }
+
+export function RNFlatList(props: Omit<FlatListProps<Item>, 'keyExtractor' | 'renderItem'>) {
+  return (
+    <FlatList {...props} keyExtractor={(item) => item.key} renderItem={({ item }) => item.label} />
```

Comment:
As a follow-up, we should do the same for DraggableFlatList before cutting next release - remove `header` in favour of just `ListHeaderComponent`.

Turns out that the following:
```
ListHeaderComponent={props.header ? () => props.header : undefined}
```
is not necessary, as we can just pass an element directly.

I believe we may have had another kind of an issue before inside Rise renderer which prompted this workaround, but since that issue is now gone, this can also be simplified.

Reply:
I'm not sure `item.label` is the best name, because this view is more than a label. Just `item` makes sense but then the implementation here would be `item.item` which seems confusing. Maybe `item.view`?

Reply:
That's fine, but would be breaking to previous FlatList users and DraggableList users if we change it! But I am down, we should simplify it